packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -354,6 +355,13 @@ export interface IGCRuntimeOptions {
 	 */
 	sessionExpiryTimeoutMs?: number;
 
+	/**
+	 * Delay between when Tombstone should run and when the object should be deleted.
+	 * This grace period gives apps a chance to intervene to recover if needed, before Sweep deletes the object.
```

Comment:
```suggestion
	 * This grace period gives a chance to intervene to recover if needed, before Sweep deletes the object.
```

Because its not just for apps but for us too.